### ISSUE TITLE
Daily cases number values are getting truncated on the border cases of Charts UI

### ISSUE THREAD

**Issue by Mohit020888** [WorldHealthOrganization/app#1460]
1. Launch the app and complete onboarding journey.
2. Tap on the Stats section in the footer.
3. Long Press on the right most graph values under different sections namely Total cases , Total deaths , Daily cases, Daily deaths

Expected, correct behavior
Actual number value corresponding to a particular day graph should be displayed clearly to the user in case of right side border values

Actual Behavior 
Actual number value corresponding to a particular day graph is getting truncated and not being displayed clearly to the user in case of right side border values

Phone/Tablet (please complete the following information):

- Device: [Huawei Honor 7c]
- OS: [Android 8.0]
- App Version: [0.11.1 + 22]

Screenshots
![Screenshot_20200823-121701](https://user-images.githubusercontent.com/63721227/91055127-dfc95d80-e641-11ea-8a8c-d9b3e8024795.jpg)
![Screenshot_20200823-121718](https://user-images.githubusercontent.com/63721227/91055129-e0fa8a80-e641-11ea-878d-823106fe6074.jpg)
![Screenshot_20200823-121606](https://user-images.githubusercontent.com/63721227/91055131-e1932100-e641-11ea-8f2c-d79c583a2a2c.jpg)
![Screenshot_20200823-121618](https://user-images.githubusercontent.com/63721227/91055133-e22bb780-e641-11ea-9353-c61e8b04a81e.jpg)

---

- [x] Searched the [existing issues](https://github.com/WorldHealthOrganization/app/issues) to ensure you are not creating a duplicate.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Issue has descriptive title focused on the feature or problem from the user's perspective

**Comment by ForgottenProgramme** [comment]
Hey, @Mohit020888 ! 
Can I please take up this issue?

**Comment by Mohit020888** [comment]
@ForgottenProgramme : thanks for taking this up ! I have assigned it to you , please create a PR and get it reviewed by @advayDev1

**Comment by ForgottenProgramme** [comment]
Hey, @advayDev1 ! I am a beginner in Flutter App Dev. 
I ran the app on emulator and I could reproduce the behavior described in the issue. 
I think I will have to tweak some code in the recent_numbers_graph.dart file to resolve the issue. Am I right?

**Comment by brunobowden** [comment]
@ForgottenProgramme - you're welcome to take a go fixing this. Just send a PR to @theswerd and @brunobowden
